ImageMarkupBuilder.js

```diff
@@ -79,7 +70,7 @@ function ImageMarkupBuilder(fabricCanvas) {
   };
 
   function applyBackground(callback) {
-    if (!isNode) {
+    if (!isNode()) {
```

Comment:
Sorry I didn't call this out before, but all the `isNode()` cases will need to be switched back now that `isNode` is a constant again. There's all the ones in this file and one in `rectangle.js`.

Reply:
Too true! I'm sorry. It was late and I accepted without enough thought.

Reply:
> Sorry I didn't call this out before, but all the `isNode()` cases will need to be switched back now that `isNode` is a constant again. There's all the ones in this file and one in `rectangle.js`.

Let's rebase in the AM